### ISSUE TITLE
Refactor Bower dependency to use the official SAP ones

### ISSUE THREAD

**Issue by Kyrodan** [saschakiefer/generator-openui5#88]
This issue is related to #73.

If I understand right:
- there exists the openui5-bower package which is _not_ an official SAP package from https://github.com/RaYell/openui5-bower
- there exists the official bower-packages from https://github.com/openui5?utf8=%E2%9C%93&query=packaged- and announced in http://openui5.tumblr.com/post/104843094047/openui5-now-also-available-via-bower

If I'm right, wouldn't it be better to include the "official" bower packages rather than the one from RaYell?

**Comment by js1972** [comment]
Yes you're right. The official SAP package came out more recently after we'd already built the generator. At some stage when we have time we'll probably switch...

On Mon, Jul 13, 2015 at 6:17 AM -0700, "Daniel Bölts" notifications@github.com wrote:

This issue is related to #73.

If I understand right:

there exists the openui5-bower package which is not an official SAP package from https://github.com/RaYell/openui5-bower

there exists the official bower-packages from https://github.com/openui5?utf8=%E2%9C%93&query=packaged-

If I'm right, wouldn't it be better to include the "official" bower packages rather than the one from RaYell?

**Comment by Kyrodan** [comment]
Ok, thanks for your reply.
I just saw that this work has already be done for the sample app.

**Comment by js1972** [comment]
Yeah I want to refactor all the templates but have been waiting for 1.30 to become generally available as there is quite a few good changes in it....Development is also moving to my js1972/ fork.  I'll write a blog post on scn when it's done but I don't expect to have time for a while yet. 

On Mon, Jul 13, 2015 at 6:53 AM -0700, "Daniel Bölts" notifications@github.com wrote:

Ok, thanks for your reply.

I just saw that this work has already be done for the sample app.
